modules/network-annotation-processor/src/main/java/org/apache/ignite/internal/network/processor/TransferableObjectProcessor.java

```diff
@@ -351,4 +354,33 @@ private void writeServiceFile(String packageName, TypeSpec registryInitializer)
             throw new ProcessingException(e.getMessage(), e);
         }
     }
+
+    /** Generates utility classes for {@link Enum} in network messages. */
+    private void generateEnumTransferableUtils(Collection<MessageClass> messageClasses, MessageGroupWrapper messageGroup) {
+        List<MessageClass> serializableMessages = collectAutoSerializableMessageClasses(messageClasses);
+
+        if (serializableMessages.isEmpty()) {
+            return;
+        }
```

Comment:
Why do we need this?

Reply:
What exactly? 
If nwtwork messages are not automatically serializable, then serializers and deserializers will not be generated for them, which means they do not need helper classes for deserializing enumerations.

Reply:
Could you please add a comment explaining this? Otherwise, it's not obvious from this part of the code.